packages/forms-web-app/src/pages/projects/register/index/_utils/is-registration-open.js

```diff
@@ -6,16 +6,50 @@ const { featureFlag } = require('../../../../../config');
 dayjs.extend(isSameOrAfter);
 dayjs.extend(isSameOrBefore);
 
-const isRegistrationOpen = (openDate, closedDate, caseRef) => {
-	const dayToday = dayjs();
-	const registrationOpenDate = dayjs(openDate);
-	const registrationClosedDate = dayjs(closedDate);
-
-	return (
-		featureFlag.openRegistrationCaseReferences.includes(caseRef) ||
-		(dayToday.isSameOrAfter(registrationOpenDate) &&
-			(dayToday.isSameOrBefore(registrationClosedDate, 'day') || !closedDate))
-	);
+const isRegistrationClosed = ({
```

Comment:
small thing, but can we have a comment here explaining that we have `isRegistrationClosed` functions specifically to check for a state where registration is closed after being opened previously and why we need that? It's a bit confusing having two separate functions that check for registration being open and closed which at first glance check the same thing

Reply:
@emil-placheta Comment added and argument renamed.